invenio_search_ui/views.py

```diff
@@ -39,7 +39,10 @@
 @blueprint.route('/search')
 def search():
     """Search page ui."""
-    return render_template(current_app.config['SEARCH_UI_SEARCH_TEMPLATE'])
+    return render_template(
+        current_app.config['SEARCH_UI_SEARCH_TEMPLATE'],
+        search_extra_params=current_app.config['SEARCH_UI_SEARCH_EXTRA_PARAMS']
```

Comment:
IMHO you have already access in jinja with ``config.SEARCH_UI_SEARCH_EXTRA_PARAMS`` :)

Reply:
yep that's correct! I will double check it and I will close it!